rebasehelper/tags.py

```diff
@@ -115,7 +159,10 @@ def sanitize(name):
                     line = line[start:].rstrip('}')  # FIXME: removing trailing braces is not very robust
                     m = tag_re.match(line)
                     if m:
-                        span = cast(Tuple[int, int], tuple(x + start for x in m.span('value')))
+                        if is_list:
+                            span = (0, len(raw_content.sections[section_index][index]) - 1)
+                        else:
+                            span = cast(Tuple[int, int], tuple(x + start for x in m.span('value')))
```

Comment:
This (and the hunk above) is just too complicated. It would be better to introduce an offset variable, that would default to 0, and was set to the length of the current prefix in case something was prepended.